docs/content/recipes/common/replace-crio-runtime.md

```diff
@@ -0,0 +1,247 @@
+        filesystem: root
+        mode: 420
+        path: /etc/crio/crio.conf.d/99-crun-runtime.conf
+EOF
+```
+
+- This is the manifest for high performance workloads setting `crun` as default runtime.
+
+```bash
+cat <<EOF > mc-crun-high-perf-default-runtime.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 60-crun-high-perf-default-runtime
+spec:
+  config:
+    ignition:
+      config:
+        replace:
+          verification: {}
+      proxy: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 3.2.0
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lPWNydW4KIyBZb3Ugd2lsbCBuZWVkIHRvIGNoYW5nZSB0aGUgQ1BVIHNldCBhc3NpZ25lZCBmb3IgSW5mcmEgbm9kZXMgZGVwZW5kaW5nIG9uIHRoZSBpbmZyYXN0cnVjdHVyZQppbmZyYV9jdHJfY3B1c2V0ID0gMC0xLDI0LTI1CgpbY3Jpby5ydW50aW1lLnJ1bnRpbWVzLmNydW5dCnJ1bnRpbWVfcGF0aCA9IC9iaW4vY3J1bgpydW50aW1lX3R5cGUgPSBvY2kKcnVudGltZV9yb290ID0gL3J1bi9jcnVuCgphbGxvd2VkX2Fubm90YXRpb25zID0gWwogICAgaW8uY29udGFpbmVycy50cmFjZS1zeXNjYWxsLAogICAgaW8ua3ViZXJuZXRlcy5jcmktby5EZXZpY2VzLAogICAgaW8ua3ViZXJuZXRlcy5jcmktby5MaW5rTG9ncywKICAgIGNwdS1sb2FkLWJhbGFuY2luZy5jcmlvLmlvLAogICAgY3B1LXF1b3RhLmNyaW8uaW8sCiAgICBpcnEtbG9hZC1iYWxhbmNpbmcuY3Jpby5pbywKXQo=
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/crio/crio.conf.d/99-crun-high-perf-runtime.conf
+EOF
+```
+
+### Creating the ConfigMap for HCP consumption
+
+This commands will create the ConfigMap with the content of the files previously generated:
+
+- The ConfigMap for regular workloads
+
+```bash
+oc create -n <hostedcluster_namespace> configmap mcp-crun-default -n clusters --from-file config=mc-crun-default-runtime.yaml
+```
+
+- The ConfigMap for high performance workloads
+
+```bash
+oc create -n <hostedcluster_namespace> configmap mcp-crun-hp-default -n clusters --from-file config=mc-crun-high-perf-default-runtime.yaml
+```
+
+### Patching the NodePool
+
+This command will patch the NodePool resource, adding the new ConfigMap created as a MachineConfig change:
+
+```bash
+oc patch -n <hostedcluster_namespace> nodepool <nodepool_name> --type=merge -p='{"spec": {"config": [{"name": "mcp-crun-default"}]}}'
+```
+
+or in the high performance case:
+
+```bash
+oc patch -n <hostedcluster_namespace> nodepool <nodepool_name> --type=merge -p='{"spec": {"config": [{"name": "mcp-crun-hp-default"}]}}'
+```
+
+
+
+## From crun to runc
+
+Assuming that your workloads suffered somekind of performance hit or something similar and you want to go back to `runc` as default runtime, this recipe will show you how to set the runc as default CRI-O runtime in your worker nodes, even after the upgrade to 4.18. This is also compatible with 4.17, if you plan to upgrade the HCP version and you want to keep the runtime as `runc`.
```

Comment:
is this disruptive?

Reply:
The got back to `runc` implies a MachineConfig change, which make the nodes to reboot.